docs/quick.rst

```diff
@@ -48,7 +51,39 @@ from a dataset in the file:
 
 For more, see :ref:`file` and :ref:`dataset`.
 
+Appendix: Creating a file
++++++++++++++++++++++++++
+
+At this point, you may wonder how :code:`mytestdata.hdf5` is created. 
+We can create a file by setting the :code:`mode` to :code:`w` when 
+the File object is initialized. Some other modes are :code:`a` 
+(for read/write/create access), and 
+:code:`r+` (for read/write access). 
+A full list of file access modes and their meanings is at :ref:`file`. ::  
+
+    >>> import h5py
+    >>> import numpy as np
+    >>> f = h5py.File("mytestfile.hdf5", "w")
 
+The :ref:`File object <file>` has a couple of methods which look interesting. One of them is ``create_dataset``, which 
+as the name suggests, creates a data set of given shape and dtype ::
+
+    >>> dset = f.create_dataset("mydataset", (100,), dtype='i')
+
+Now we can flush the file to the disk
```

Comment:
Flushing the file is unnecessary here.  Also, "f.close()" is the canonical way to close a file, since it will succeed even if other objects are still open.